test/integration/logger.tap.js

```diff
@@ -49,3 +52,20 @@ tap.test('logger', function (t) {
     })
   })
 })
+
+tap.test('Logger output', (t) => {
+  t.autoend()
+
+  t.test('Check for logger output at debug level', async (t) => {
+    const { stdout, stderr } = await exec('node -r ../../../index.js hello.js', {
```

Comment:
very creative way of testing.

Reply:
Maybe another ticket but also providing one that calls `--experimental-loader` and asserts that since those are the two big use cases

Reply:
I decided to just do it here, added another test as suggested.